book/.vscode/ltex.dictionary.en-US.txt

```diff
@@ -0,0 +1,27 @@
+GObject
```

Comment:
You probably want to drop/keep the vscode specific files locally

Reply:
Ah yes, we can get rid of all my weird stuff including gitlab before we merge it if that's what you prefer